etna/text_formats/blocks.py

```diff
@@ -0,0 +1,13 @@
+from wagtail.core import blocks
```

Comment:
I really like the idea of using a separate app for rich text fields, it helps reduce the need for the dreaded `base` app too!

I think the name `text_formats` might be a bit restrictive though. In the past, we've written rich text converters, editor features as well as supporting [legacy rich text](https://docs.wagtail.io/en/v2.12.4/reference/contrib/legacy_richtext.html) and it'd be nice to keep this all in one place.

Wagtail is a bit inconsistent with its naming here, using both [`rich_text`](https://github.com/wagtail/wagtail/tree/main/wagtail/admin/rich_text ) and [`richtext`](https://github.com/wagtail/wagtail/tree/main/wagtail/contrib/legacy/richtext) but something along those lines would be good.

Reply:
I've renamed the app `richtexts`, to contain any variations that may be required.

Reply:
I've removed the richtext(s) app as it's not needed using the features defined in settings as we discussed.
I might need to add it back in later for blocks requiring different feature sets. But I will define those sets in settings as well.